packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5734,7 +5143,7 @@ export default class Meeting extends StatelessWebexPlugin {
 
             // eslint-disable-next-line func-names
             // eslint-disable-next-line prefer-arrow-callback
-            if (this.type === _CALL_) {
+            if (this.type === _CALL_ || this.meetingState === FULL_STATE.ACTIVE) {
```

Comment:
what is FULL_STATE.ACTIVE?

Reply:
yeah, this just saves 1s on addMedia() call because previously we were always starting the joiningTimer (line 5149) and checking the state after 1s, but the state might be already ACTIVE so no need to start the timer in that case. This is especially true in all unit tests - all addMedia() tests used to take at least 1s, now they all finish in a few milliseconds.